todo_app/app.py

```diff
@@ -30,73 +30,61 @@ def index():
     @app.route('/new_item', methods=['POST'])
     def new_item():
         new_item_title = request.form.get('new_item_title')
-        trello_default_list = get_trello_list_id('To Do')
+        default_list = 'todo'
         if request.form.get('new_item_due'):
             due_date = datetime.datetime.strptime(request.form.get('new_item_due'), '%Y-%m-%d')
         else:
-            due_date = datetime.date.today() + datetime.timedelta(30)
+            due_date = datetime.datetime.today() + datetime.timedelta(30)
         
         description = request.form.get('new_item_description')
         
-        new_card = TrelloCard(0, new_item_title, trello_default_list, due_date, description, datetime.datetime.today())
-        create_trello_card(new_card)
+        new_card = ToDoCard(0, new_item_title, default_list, due_date, description, datetime.datetime.today())
+        create_todo_card(new_card)
         return redirect(request.headers.get('Referer'))
 
     @app.route('/remove_item', methods=['POST'])
     def remove_existing_item():
-        trello_list_id = {}
-        trello_list_id['To Do'] = get_trello_list_id('To Do')
-        trello_list_id['Doing'] = get_trello_list_id('Doing')
-        trello_list_id['Done'] =get_trello_list_id('Done')
-        allcards = get_trello_cards()
-        all_lists = get_trello_lists_on_board()
+        allcards = get_todo_cards()
+        lists = {'todo','doing','done'}
         
         toggle_item = request.form.get('delete_id')
 
         for card in allcards:
-            if card.id == toggle_item:
-                
-                delete_trello_card(card.id)
+            if toggle_item == str(card.id):
+                delete_todo_card(card.id)
         
         return redirect(request.headers.get('Referer'))
 
 
     @app.route('/Doing', methods=['POST'])
     def in_progress():
-        trello_list_id = {}
-        trello_list_id['To Do'] = get_trello_list_id('To Do')
-        trello_list_id['Doing'] = get_trello_list_id('Doing')
-        trello_list_id['Done'] =get_trello_list_id('Done')
-        
-        allcards = get_trello_cards()
-        all_lists = get_trello_lists_on_board()
-        
+        allcards = get_todo_cards()
+        lists = {'todo','doing','done'}
+
         toggle_item = request.form.get('item_id')
 
         for card in allcards:
-            if card.id == toggle_item:
-                new_list_id = trello_list_id['Doing']
-                move_trello_card(card.id, new_list_id)
+            if toggle_item == str(card.id):
+                for l in lists:
+                    if l == "doing":
+                        desired_list_id = l
+                        move_todo_card(card.id, desired_list_id)
```

Comment:
Isn't this for-loop equivalent to:

```suggestion
            move_todo_card(card.id, 'doing')
```

Since `lists` is a hardcoded list that always contains 'doing'.